docs/tutorials/first-run-experience-tutorial.md

```diff
@@ -0,0 +1,152 @@
+---
+title: First-run experience tutorial
+description: Learn how to implement a first-run experience for your Office Add-in.
+ms.date: 08/09/2024
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Build an Office Add-in with a basic first-run experience
+
+In this article, you'll walk through the process of updating a task pane add-in to include a [first-run experience](../design/first-run-experience-patterns.md) using the value placemat pattern. When the user runs the add-in, the add-in determines whether or not to show the first-run experience by checking [local storage](../develop/persisting-add-in-state-and-settings.md#browser-storage) for the existence of a certain flag.
+
+This tutorial provides instructions and screenshots for Excel but you can use a similar pattern to implement a first-run experience in other Office applications where Office Web Add-ins are supported. Also note that actual interactions with your selected Office application will differ.
```

Comment:
> Also note that actual interactions with your selected Office application will differ.
I'm not sure what this means in this context. Is the code different or the UI flow?

Reply:
@AlexJerabek We add an instruction for the user to select cells before clicking Run. I was thinking about a situation like that where it's only applicable to Excel. That said, I'll remove that sentence since I have a step that asks the dev to update the instruction